python-package/rgf/utils.py

```diff
@@ -412,17 +435,33 @@ def cleanup(self):
             Returns the number of removed files.
         """
         n_removed_files = 0
-        if self._estimators is not None:
+        if hasattr(self, '_estimators'):
             for est in self._estimators:
                 n_removed_files += cleanup_partial(est._file_prefix,
                                                    remove_from_list=True)
 
         # No more able to predict without refitting.
-        self._fitted = None
+        self._fitted = False
         return n_removed_files
 
     def _get_params(self):
-        raise NotImplementedError(NOT_IMPLEMENTED_ERROR_DESC)
+        return {'config': CONFIG}
+
+    @classmethod
+    def _set_config(cls):
+        global CONFIG
+        if CONFIG is None:
+            CONFIG = Config()
```

Comment:
Another option is to use singleton pattern for config.
Because only one `Config` instance can exist.

Reply:
Yeah, let's try to rework it in the future.